students/zelichowski_michal/lesson_02_flow_control/chocolate_bar.py

```diff
@@ -0,0 +1,21 @@
+"""Chocolate bar has the form of a rectangle divided into n×m portions.
+Chocolate bar can be split into two rectangular parts by breaking it
+along a selected straight line on its pattern. Determine whether it
+is possible to split it so that one of the parts will have exactly k
+squares.
+
+The program reads three integers: n, m, and k. It should print
+YES or NO. """
+
+# Read an integer:
+n = int(input())
+m = int(input())
+k = int(input())
```

Comment:
Podążasz za wytycznymi z zadania na snakify odnośnie nazw zmiennych i to jest ok.
Niemniej ja bym tutaj poszalał i zmienił nazwy zmiennych, żeby nie musieć wracać do treści zadania kiedy chcę się dowiedzieć do czego one służą.
Zrobiłeś zgodnie z poleceniem, więc nie poprawiaj.